source/routing/raptor_api.h

```diff
@@ -35,8 +35,11 @@ www.navitia.io
 #include "utils/flat_enum_map.h"
 #include "type/rt_level.h"
 #include <limits>
+#include "raptor.h"
 #include "routing/routing.h"
 
+#define MAX_NB_RAPTOR_CALL 100
```

Comment:
minor: since it's only used in .cpp, for a better encapsulation, shouldn't this be in .cpp? and const static int ?